plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -3454,6 +3448,8 @@ public boolean isCentosHost() {
             return DiskDef.DiskBus.IDE;
         } else if (platformEmulator.startsWith("Other PV Virtio-SCSI")) {
             return DiskDef.DiskBus.SCSI;
+        } else if (isUefiEnabled && platformEmulator.startsWith("Windows")) {
+            return DiskDef.DiskBus.SATA;
```

Comment:
Is this only required in case of UEFI secure mode only?

Reply:
this is required for both secure/legacy UEFI boot type on Windows machines